src/components/editor/ArticleController.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class ArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleCategorySubmit = this.handleCategorySubmit.bind(this);
+    this.handleAuthorDelete = this.handleAuthorDelete.bind(this);
+    this.handleAuthorAdd = this.handleAuthorAdd.bind(this);
+    this.handleDescriptionSubmit = this.handleDescriptionSubmit.bind(this);
+    this.handleIssueSubmit = this.handleIssueSubmit.bind(this);
+  }
+
+  static getFalcorPath(params) {
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+  }
+
+  handleCategorySubmit(event) {
+    event.preventDefault();
+  }
+
+  handleAuthorDelete(event) {
+    event.preventDefault();
+  }
+
+  handleAuthorAdd(event) {
+    event.preventDefault();
+  }
+
+  handleDescriptionSubmit(event) {
+    event.preventDefault();
+  }
+
+  handleIssueSubmit(event) {
+    event.preventDefault();
+  }
+
+  render() {
+    if (this.state.ready) {
+      if (!this.state.data) {
+        return <div><p>No articles match this slug</p></div>;
+      }
+
+      let slug = this.props.params.slug;
+      let data = this.state.data.articlesBySlug[slug];
+
+      return (
+        <div>
+          <h3>{data.title}</h3>
+          <p>The default values are the current values for the article.
+          Change them and press submit to update the values.</p>
+          <form onSubmit={this.handleCategorySubmit}>
```

Comment:
So here you mean mainly getting rid of the form tags?

I'll play with removing the form tags and using <button>'s instead of submits, and change the <br/> 's